checkClock/app/helpers/branches_helper.rb

```diff
@@ -0,0 +1,2 @@
+module BranchesHelper
+end
```

Comment:
If you don't use this file, you can delete it.

Reply:
This file was deleted in a new commit.